javascript/js-lowerings/src/org/jetbrains/dukat/js/lowerings/lowerings.kt

```diff
@@ -0,0 +1,145 @@
+package org.jetbrains.dukat.js.lowerings
+
+import org.jetbrains.dukat.logger.Logging
+import org.jetbrains.dukat.astCommon.IdentifierEntity
+import org.jetbrains.dukat.astModel.*
+import org.jetbrains.dukat.astModel.modifiers.VisibilityModifier
+import org.jetbrains.dukat.astModel.statements.StatementModel
+import org.jetbrains.dukat.js.declarations.toplevel.JSFunctionDeclaration
+import org.jetbrains.dukat.js.declarations.JSModuleDeclaration
+import org.jetbrains.dukat.js.declarations.misc.JSParameterDeclaration
+import org.jetbrains.dukat.js.declarations.toplevel.JSClassDeclaration
+import org.jetbrains.dukat.js.declarations.toplevel.JSReferenceDeclaration
+import org.jetbrains.dukat.js.declarations.toplevel.JSDeclaration
+import org.jetbrains.dukat.translator.ROOT_PACKAGENAME
+
+
+class JSModuleFileLowerer(private val moduleDeclaration: JSModuleDeclaration) {
+
+    private val logger = Logging.logger("Lowering")
+
+    private val moduleName = IdentifierEntity(moduleDeclaration.moduleName)
+
+    private val MODULE_ANNOTATION = AnnotationModel(
+            name = "JsModule",
+            params = listOf(moduleName)
+    )
+
+    private val ANY_NULLABLE_TYPE = TypeValueModel(
+            value = IdentifierEntity("Any"),
+            params = emptyList<TypeParameterModel>(),
+            metaDescription = null,
+            nullable = true
+    )
+
+    private fun JSParameterDeclaration.convert(): ParameterModel {
+        return ParameterModel(
+                name = name,
+                type = ANY_NULLABLE_TYPE,
+                initializer = null,
+                vararg = vararg,
+                optional = false
+        )
+    }
+
+    private fun JSFunctionDeclaration.convert(): FunctionModel {
+        val parameterModels = mutableListOf<ParameterModel>()
+
+        for(jsParameter in parameters) {
+            parameterModels.add(jsParameter.convert())
+        }
```

Comment:
we can just val parameterModels = paramaters.map { it.convert() }.toMutableList()